src/mount.ts

```diff
@@ -133,6 +137,46 @@ export function mount(originalComponent: any, options?: MountingOptions) {
   const { emitMixin, events } = createEmitMixin()
   vm.mixin(emitMixin)
 
+  // stubs
+  if (options?.global?.stubs) {
+    transformVNodeArgs((args) => {
```

Comment:
Maybe we can destructure like this, so its easier to understand context later down?
```suggestion
    transformVNodeArgs(([componentOptions]) => {
```

Reply:
Thanks for the suggestion, but I am not really a fan of this refactor. This variable can be a string (eg 'div'), or an object of options, so componentOptions is a bit misleading. I guess the ideal name is `htmlTagOrComponentOptions`. I will just put a comment here for now.